### ISSUE TITLE
AR not working!

### ISSUE THREAD

**Issue by EdwardMoyse** [HSF/phoenix#418]
![31250184-870F-47C1-A559-B1B166A54E22](https://user-images.githubusercontent.com/6764617/163187425-24ba8541-cabb-4f4c-bee3-d411aa286ee2.png)

Tested on iPhone with Safari and Firefox. 
Samsung with native browser and chrome

**Comment by EdwardMoyse** [comment]
Okay, I thought that WebKit finally had support for WebXR, but apparently it's still "In Development"
https://webkit.org/status/#specification-webxr

Still, I would have expected it to work on Android with Chrome.

**Comment by 9inpachi** [comment]
It only worked on a special browser in iOS if I remember? https://github.com/HSF/phoenix/issues/212#issuecomment-874584680

**Comment by EdwardMoyse** [comment]
Yes, sorry I wasn't explicit enough but this is what I meant in my comment above. But as I said, I would have expected it to work on Android. Maybe something has to be enabled on the phone? But if so, we should have a pop up explaining this because most people will just assume AR doesn't work yet.

**Comment by 9inpachi** [comment]
Seems to work fine in Chrome on my Android. Xiaomi Poco F3.

Does the Samsung you tested on support AR?
